docs/oauth2.md

```diff
@@ -43,7 +43,7 @@ first.
 
 Hydra uses the [JSON Web Key Manager](./jwk.md) to retrieve the
 key pair `hydra.openid.id-token` for signing ID tokens. You can use that endpoint to retrieve the public key for verification,
-has Hydra is not supporting OpenID Connect Discovery yet.
+as Hydra does not supporting OpenID Connect Discovery yet.
```

Comment:
In fact, this is no longer correct as Hydra does indeed support OpenID Connect Discovery :) Could you change this to

> public key for verification. Additionally, Hydra supports OpenID Connect Discovery.